### PR TITLE
[plugin.video.cbcolympics] 1.0.0

### PR DESCRIPTION
**Pull request by fourpointsix**
### Description
This addon is for browsing and watching CBC's 2018 Winter Olympics streams. The streams are geo-blocked so this is generally a Canada-only addon.
### Checklist:

- [X] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [X] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-plugins/blob/master/CONTRIBUTING.md) document
- [X] Each add-on submission should be a single commit with using the following style: [plugin.video.foo] v1.0.0

Additional information :
- Submitting your add-on to this specific branch makes it available to any Kodi version equal or higher than the branch name with the applicable Kodi dependencies limits.
- [add-on development](http://kodi.wiki/view/Add-on_development) wiki page.
- Kodi [pydocs](http://kodi.wiki/view/PyDocs) provide information about the Python API
- [PEP8](https://www.python.org/dev/peps/pep-0008/) codingstyle which is considered best practice but not mandatory.
- This add-on repository has automated code guideline check which could help you improve your coding. You can find the results of these check at [Codacy](https://www.codacy.com/app/Kodi/repo-plugins/dashboard). You can create your own account as well to continuously monitor your python coding before submitting to repo.
- Development questions can be asked in the [add-on development](http://forum.kodi.tv/forumdisplay.php?fid=26) section on the Kodi forum.

### CODE REVIEW

**Comment by fourpointsix** [comment]
Clearly I messed up this request. I tried to request a commit to the Krypton branch but it appears to have pulled everything. I'll close this.

**Comment by Rechi** [comment]
changed the base branch for you

**Comment by fourpointsix** [comment]
Thank you for the assistance but I only intended to pull the 9 files in the plugin.video.cbcolympics directory. This request has 2000 files in it so isn't it horribly wrong? I've spent all day trying to understand how to Git properly and I can't seem to figure it out.

**Comment by Rechi** [comment]
It is already correct now, only mistake you have made, was selecting master branch instead of krypton during creating the pull request.

**Comment by fourpointsix** [comment]
Oh I see now, thank you! I had to refresh the page before it looked correct.

File: plugin.video.cbcolympics/changelog.txt [hunk 1]
```diff
@@ -0,0 +1,2 @@
+v1.0.0
+- Initial version for 2018 Winter Olympics
```
Comment (Lunatixz) [inline plugin.video.cbcolympics/changelog.txt]: FYI changelogs are no longer used, they were replaced with the <news> parameter in addon.xml.
https://kodi.wiki/view/Addon.xml#.3Cnews.3E

Comment (MartijnKaijser) [reply plugin.video.cbcolympics/changelog.txt]: you can still keep changelogs for yourself of course

Comment (Lunatixz) [reply plugin.video.cbcolympics/changelog.txt]: @Rechi LGTM

Comment (Lunatixz) [reply plugin.video.cbcolympics/changelog.txt]: @MartijnKaijser True, Just encouraging the use of <news> =)

**Comment by Lunatixz** [comment]
@fourpointsix are you aware this plugin is in TVaddons repository?

**Comment by Lunatixz** [comment]
@Rechi are your concerns resolved?

**Comment by Rechi** [comment]
I never had any. PR was opened against master instead of Krypton, which is the only thing I've done here.

File: plugin.video.cbcolympics/changelog.txt (hunk 1 above)
Comment (fourpointsix) [reply plugin.video.cbcolympics/changelog.txt]: Good to know. I've updated my copy with <news> so the next revision will have it.

**Comment by fourpointsix** [comment]
@Lunatixz Yes I am - they asked me beforehand. Thanks for the merge. I assume that I'd submit a future revision as a new pull request, correct?

**Comment by Lunatixz** [comment]
@fourpointsix Yes just issue a new pull for future updates.

**Pull request merged**